tarea1/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tarea 1</title>
+</head>
+<body>
+    <form>
+        <label for="salario-anual">Salario Anual:</label>
+        <input type="text" id="salario-anual">
+        <label for="salario-mensual">Salario Mensual:</label>
+        <input type="text" disabled id="salario-mensual">
+        <button id="calcular-salario-mensual">Calcular</button>
+        <button type="reset">Limpiar</button>
```

Comment:
En este caso me parece que queda mejor un input
```suggestion
        <input type="reset" value="Limpiar">
```